### ISSUE TITLE
Emoji + Nerd Font Support?

### ISSUE THREAD

**Issue by BlueFalconHD** [charmbracelet/lipgloss#160]
I would love to see support for double width characters and also having functions to fetch emojis and nerd font icons by name, to help declutter all of the unicode symbols. This would also create an opportunity to have alternative icons when no double width characters are supported.

**Comment by meowgorithm** [comment]
NB: double width characters are supported in all Charm products as a rule.

**Comment by BlueFalconHD** [comment]
> NB: double width characters are supported in all Charm products as a rule.

Great! The second part of what I said is the main point here. I haven't really experimented with emoji yet in bubbletea, but I will try it out.

**Comment by caarlos0** [comment]
Per my understanding this is resolved. Please let me know otherwise.

Cheers!

**Comment by meowgorithm** [comment]
Yeah, generally speaking I would leave the second portion as an exercise to the user. @BlueFalconHD if you disagree, just let us know.
